### PR TITLE
add support for healthcare datasets

### LINKED ISSUE #230: Support healthcare API
https://github.com/terraform-google-modules/terraform-google-healthcare

### PR DESCRIPTION
**Pull request by umairidris**
Fix #230

### CODE REVIEW

File: templates/tfengine/components/project/healthcare_datasets/main.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  source  = "terraform-google-modules/healthcare/google"
+  version = "~> 1.0.0"
+
+  name     = "{{.name}}"
+  project  = var.project_id
+  location = "{{get . "location" $.healthcare_location}}"
+
+  {{hclField . "iam_members" false}}
+
+  {{if has . "dicom_stores" -}}
```
Comment (MartinPetkov) [inline templates/tfengine/components/project/healthcare_datasets/main.tf:26]: "dicom_stores" should be in the example, or is it mutually exclusive from fhir_stores?

File: templates/tfengine/components/project/healthcare_datasets/main.tf [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {{range .fhir_stores -}}
+    {
+      name = "{{.name}}"
+      {{hclField . "iam_members" false -}}
+    }
+    {{- end}}
+  ]
+  {{- end}}
+
+  {{- if has . "hl7_v2_stores"}}
```
Comment (MartinPetkov) [inline templates/tfengine/components/project/healthcare_datasets/main.tf:48]: Same here, should be in the example or mentioned as an option.

File: templates/tfengine/components/project/healthcare_datasets/main.tf (hunk 1 above)
Comment (umairidris) [reply templates/tfengine/components/project/healthcare_datasets/main.tf:26]: Done.

File: templates/tfengine/components/project/healthcare_datasets/main.tf (hunk 2 above)
Comment (umairidris) [reply templates/tfengine/components/project/healthcare_datasets/main.tf:48]: Done.

**Pull request merged**